src/core/task/tools/handlers/ApplyPatchHandler.ts

```diff
@@ -685,7 +685,10 @@ export class ApplyPatchHandler implements IFullyManagedTool {
 
 		if (shouldAutoApprove) {
 			await config.callbacks.removeLastPartialMessageIfExistsWithType("ask", "tool")
-			await config.callbacks.say("tool", completeMessage, undefined, undefined, false)
+			const sayTs = await config.callbacks.say("tool", completeMessage, undefined, undefined, false)
```

Comment:
Is this something we can replace with the new ToolResultUtils.cleanupAndSendToolMessage method?